tests/brun/path-4.txt

```diff
@@ -0,0 +1,3 @@
+brun -n -c '1' '(((0x1337 . (0x1337 . (42 . 0x1337))) . 0x1337) . 0x1337)'
+cost = 53
+(((4919 4919 42 . 4919) . 4919) . 4919)
```

Comment:
You may want to note that we print this in a (possibly) unintuitive way. I was thrown off at least. I believe it's consistent though.

Reply:
Ya, I know what you mean. To some extent, it can't be helped, given the syntax that lisp chose.